src/pages/{StoreCollection.slug}.tsx

```diff
@@ -13,6 +14,16 @@ import type {
   CollectionPageQueryQuery,
   CollectionPageQueryQueryVariables,
 } from '@generated/graphql'
+import Breadcrumb from 'src/components/ui/Breadcrumb'
+import type { BreadcrumbProps } from 'src/components/ui/Breadcrumb'
+import ScrollToTopButton from 'src/components/ui/ScrollToTopButton'
+
+import '../styles/pages/index.scss'
```

Comment:
hmm, we are reusing the styles from the homepage here. I wonder if we could move the common styles and import them into each proper style page file afterward.

Reply:
It's going to be addressed in [this other PR](https://github.com/vtex-sites/base.store/pull/292).